src/pages/docs.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react';
+import { DOCUMENTATION_PATH, LATEST_ABLY_API_VERSION_STRING } from '../../data/transform/constants';
+import { HomepageContent, Section } from '../components/Homepage/HomepageContent';
+import Layout from '../components/Layout';
+import { LeftSideBar } from '../components/StaticQuerySidebar';
+import { graphql } from 'gatsby';
+import { Helmet } from 'react-helmet';
+
+type MetaData = {
+  title: string;
+  description: string;
+  image: string;
+  twitter: string;
+};
+
+const IndexPage = ({
+  data: {
+    pageContentYaml: { sections, meta },
+  },
+}: {
+  data: { pageContentYaml: { sections: Section[]; meta: MetaData } };
+}) => {
+  const openGraphTitle = sections[0].title;
```

Comment:
`sections[0]?.title` - I know it is unlikely to happen but it could happen by mistake and would be nice to make it tolerant.